drivers/overlay/overlay.go

```diff
@@ -142,8 +142,9 @@ func Init(home string, options graphdriver.Options) (graphdriver.Driver, error)
 	if opts.mountProgram == "" {
 		switch fsMagic {
 		case graphdriver.FsMagicAufs, graphdriver.FsMagicZfs, graphdriver.FsMagicOverlay, graphdriver.FsMagicEcryptfs:
-			logrus.Errorf("'overlay' is not supported over %s", backingFs)
-			return nil, errors.Wrapf(graphdriver.ErrIncompatibleFS, "'overlay' is not supported over %s", backingFs)
+			msg := fmt.Sprintf("'overlay' is not supported over %s, a mount_program is required", backingFs)
+			logrus.Errorf(msg)
```

Comment:
Non-blocking: I prefer to only log when an error is not returned. Callers should log the error instead.

Reply:
Removed this line, although there are other instances.  We could remove those in a future PR.